fetchcode/api.py

```diff
@@ -26,29 +27,45 @@ class Response:
 - `content_type`: content type of the file
 - `size`: size of the retrieved content in bytes
 - `url`: fetched URL
+- `scheme` : scheme of the URL
     """
-    def __init__(self, location, content_type, size, url):
+    def __init__(self, location, content_type, size, url, scheme):
         self.location = location
         self.content_type = content_type
         self.size = size
         self.url = url
- 
+        self.scheme = scheme
+
 
-def fetch(url):
+def fetch(url, location=None):
```

Comment:
What is location about?

Reply:
If user wants to give a customised location of its own choice, while using this library, they can use this location parameter.

Reply:
OK, then explain that in the function docstring

Reply:
Got this one